package.json

```diff
@@ -9,7 +9,7 @@
   "types": "./dist/index.d.ts",
   "sideEffects": false,
   "engines": {
-    "node": ">=18"
+    "node": ">=10"
```

Comment:
I guess something went wrong with rebasing? 🤓

Reply:
Storybook was not working for me with Node 18, so I was temporarily using a lower version (and committed by mistake). Will take a look and open a PR if I have some time.